apis/python/test/test_distance_metrics.py

```diff
@@ -0,0 +1,163 @@
+import os
+
+import numpy as np
+import pytest
+from array_paths import *
+from sklearn.neighbors import NearestNeighbors
+
+from tiledb.vector_search import _tiledbvspy as vspy
+from tiledb.vector_search.ingestion import ingest
+from tiledb.vector_search.utils import load_fvecs
+
+siftsmall_uri = siftsmall_inputs_file
+queries_uri = siftsmall_query_file
```

Comment:
(Minor) Can we not do this and instead reference `siftsmall_inputs_file` directly? It's what other tests do, so nice to be consistent.